### ISSUE TITLE
Problem with webpack v3.8.0

### ISSUE THREAD

**Issue by mrk-j** [JeffreyWay/laravel-mix#1279]
- Laravel Mix Version: 1.5.0
- Node Version: v6.11.4
- NPM Version: 3.10.10 
- OS: macOS Sierra 10.12.6 (16G29)

### Description:
An hour ago webpack v3.8.0 was released, with the following bugfix: `fix schema for stats and be more strict`. When running  `npm run production` I get this error:

```
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.stats should be one of these:
   object { context?, hash?, version?, timings?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? } | boolean | "none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose"
   Used by the webpack CLI program to pass stats options.
   Details:
    * configuration.stats has an unknown property 'errors'. These properties are valid:
      object { context?, hash?, version?, timings?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? }
    * configuration.stats has an unknown property 'warnings'. These properties are valid:
      object { context?, hash?, version?, timings?, assets?, env?, colors?, maxModules?, chunks?, chunkModules?, modules?, children?, cached?, cachedAssets?, reasons?, source?, warningsFilter?, excludeAssets?, excludeModules?, exclude?, entrypoints?, errorDetails?, chunkOrigins?, modulesSort?, moduleTrace?, chunksSort?, assetsSort?, publicPath?, providedExports?, usedExports?, optimizationBailout? }
    * configuration.stats should be a boolean.
    * configuration.stats should be one of these:
      "none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose"
```

This is fixed by removing the `errors` and `warnings` properties in `src/builder/webpack-default.js`. For now I fixed the issue by adding the following dependency to my own `package.json` file:

```
"webpack": "3.5.0",
```

### Steps To Reproduce:
1. `npm install laravel-mix --save-dev` 
2. Add [NPM scripts](https://github.com/JeffreyWay/laravel-mix/blob/master/docs/installation.md)
3. `touch webpack.mix.js`
4. `npm run production`

**Comment by Lorempn2** [comment]
Same issue here.

**Comment by MattPurland** [comment]
Damn, you guys beat me to it! I've been pulling my hair out for the last hour to find that 3.8.0 got released just an hour ago >.<

**Comment by AlexThomas90210** [comment]
I have the same problem

**Comment by Lorempn2** [comment]
Hardcoding webpack in your own package.json to, for example:
`"webpack": "3.7.0"`, is the fix for now.

**Comment by h1884387** [comment]
same problem here :( "^3.5.0"` in `package.json` feels maybe to relaxed, i would go with `3.5.*` and so on

**Comment by aliukevicius** [comment]
Same issue. @Lorempn2 thanks for the solution.

**Comment by ruchern** [comment]
Webpack was just released. Like literally. Give Jeffrey some time to update at least.

**Comment by ruchern** [comment]
@Lorempn2 Either that or just reset your git to the last working state of `Webpack 3.7.0` and run `npm install` or `yarn`

**Comment by rafito** [comment]
same problem here

**Comment by zimonline** [comment]
Same here.

**Comment by JeffreyWay** [comment]
I'm on it, everyone. Will get a new release out shortly.

**Comment by JeffreyWay** [comment]
Fixed in bdc93507e19809602337e3a30da11bf880e86c7a

Tagged as 1.5.1, so you can do a `npm laravel-mix`.

**Comment by mrk-j** [comment]
It seems they also fixed this from the webpack side: https://github.com/webpack/webpack/pull/5840
